wiremock/src/integTest/java/com/cognifide/aem/stubs/wiremock/HelloWorldTest.java

```diff
@@ -61,12 +61,22 @@ public void shouldGetFineWithBody() {
   }
 
   @Test
-  public void shouldGetFineWithBodyTemplated() {
+  public void shouldGetFineWithBodyTemplatedByPebble() {
```

Comment:
pebble should stay only if admin mappings listing / serialization will work / not fail